src/overlays/actors/ovl_Dm_Opstage/z_dm_opstage.h

```diff
@@ -3,16 +3,19 @@
 
 #include "global.h"
 
+#define DMOPSTAGE_GET_TYPE(thisx) (((thisx)->params & 0xFF))
+
 struct DmOpstage;
 
 typedef void (*DmOpstageActionFunc)(struct DmOpstage*, GlobalContext*);
-
 typedef struct DmOpstage {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x5C];
-    /* 0x01A0 */ DmOpstageActionFunc actionFunc;
-    /* 0x01A4 */ char unk_1A4[0xC];
-} DmOpstage; // size = 0x1B0
+    /* 0x000 */ DynaPolyActor dyna;
+    /* 0x15C */ UNK_TYPE1 pad_15C[0x44];
+    /* 0x1A0 */ DmOpstageActionFunc actionFunc;
+    /* 0x1A4 */ f32 posX;
+    /* 0x1A8 */ f32 posY;
+    /* 0x1AC */ f32 posZ;
```

Comment:
```suggestion
    /* 0x1A4 */ Vec3f pos;
```
Not required but I think this is cleaner.